flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -169,4 +186,113 @@ public ExecutionConfig disableObjectReuse() {
 	public boolean isObjectReuseEnabled() {
 		return objectReuse;
 	}
+
+	// --------------------------------------------------------------------------------------------
+	//  Registry for types and serializers
+	// --------------------------------------------------------------------------------------------
+
+	/**
+	 * Registers the given Serializer as a default serializer for the given type at the
```

Comment:
I think it would be better to call this method `registerTypeWithSerializer`.
defaultSerializers are something different in Kryo.

I'm sure users who know Kryo are confused by this "Registers the given Serializer as a default serializer for the given type at the"